docs/src/global/example.jsx

```diff
@@ -0,0 +1,127 @@
+import {
+    Button,
+    Grid,
+    Icon,
+} from 'react-cm-ui';
+/* eslint-disable import/extensions */
+import Collapse from 'react-cm-ui/utils/collapse';
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+/* eslint-enable import/extensions */
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Highlighter from './highlighter';
+
+const propTypes = {
+    children: PropTypes.node.isRequired,
+    rawCode: PropTypes.string.isRequired,
+};
+
+const useStyles = makeStyles((theme) => {
+    const {
+        palette,
+        shape,
+        typography,
+    } = theme;
+
+    return {
+        codeContainerColumn: {
+            padding: '11px 0',
+        },
+        codeToggleIcon: {
+            color: (props) => (
+                props.isCodeExpanded ?
+                    palette.text.active :
+                    palette.text.secondary
+            ),
+            fontWeight: typography.fontWeightBold,
+        },
+        exampleColumn: {
+            borderRadius: shape.borderRadius,
+            boxShadow: `0 0 0 1px ${palette.border.secondary}`,
+            padding: 22,
+        },
+        highlighter: {
+            marginTop: '11px !important',
+        },
+        root: {
+            margin: '0 !important',
+            '& .highlighter.block': {
+                marginBottom: '11px !important',
+            },
+        },
+        sourceButton: {},
+    };
+});
+
+function Example(props) {
+    const {
+        children,
+        rawCode,
+    } = props;
+    const [isCodeExpanded, setIsCodeExpanded] = useState(false);
+
+    const onCodeToggle = () => {
+        setIsCodeExpanded(!isCodeExpanded);
+    };
+
+    const classes = useStyles({
+        ...props,
+        isCodeExpanded,
+    });
+    const codeButtonTitle = 'Show Source';
+
+    return (
+        <Grid
+            className={classes.root}
+            columns={1}
+        >
+            <Grid.Row>
+                <Grid.Column
+                    className={classes.exampleColumn}
+                >
+                    {children}
+                </Grid.Column>
+            </Grid.Row>
+
+            <Grid.Row>
+                <Grid.Column
+                    className={classes.codeContainerColumn}
+                    textAlign="right"
+                >
+                    <div>
+                        <Button
+                            className={classes.sourceButton}
+                            color="transparent"
+                            icon
+                            onClick={onCodeToggle}
+                            title={codeButtonTitle}
+                        >
+                            <span
+                                className={classes.codeToggleIcon}
+                            >
+                                {'</>'}
```

Comment:
Funny, I don't remember doing that.

Reply:
OH! Nope, I definitely remember implementing that.

Reply:
It's the source button to toggle the expand/collapse of the source code below the example.

Reply:
Right, I got that.  But I think ESLINT doesn't like `{'Some string'}`.  I feel like I've had it yell at me about that (which in HC is good, because it encourages you to use `i18n()`).

Reply:
Oh, gotcha, you are right.

Reply:
Resolving per Team's phone conversation with @groberts314.